IPs/RSKIP414.md

```diff
@@ -46,21 +46,22 @@ TBD
 
 ### Proposed RSKIPs
 
-- [RSKIP-398](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP398.md): PUSH0 opcode
-- [RSKIP-400](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP400.md): Calldata gas cost reduction
-- [RSKIP-412](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP412.md): BASEFEE opcode
-- [RSKIP-203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md): GetCallStackDepth precompiled contract
+- [RSKIP-398](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP398.md): `PUSH0` opcode
+- [RSKIP-400](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP400.md): `Calldata` gas cost reduction
+- [RSKIP-412](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP412.md): `BASEFEE` opcode
+- [RSKIP-203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md): `GetCallStackDepth` precompiled contract
```

Comment:
```suggestion
- [RSKIP-203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md): `getCallStackDepth` precompiled method
```
Also here. The RSKIP says `method` instead of `contract` and the name starts with lower case